runtime/neumann/src/lib.rs

```diff
@@ -110,6 +110,7 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
+	parachain_staking::InitGenesisOnRuntimeUpgrade<Runtime>,
```

Comment:
This migrations attempts to replicate some of the storage defaults that would have been set if this pallet had been present at genesis.

Reply:
when do we need to remove this? when we do the next upgrade?

Reply:
yah from what i've seen, once we're sure this is in all of the chains correctly, other chains remove their migration

Reply:
Ideally we remove it before the next upgrade. Even if we don't this migration won't cause any harm by being run more than once.

Reply:
Removed this in favor of using the on runtime hook in the pallet